bookingmanagerpersistence/src/main/java/cz/fi/muni/pa165/entity/Reservation.java

```diff
@@ -27,43 +33,141 @@
     public Reservation() {
     }
 
-    public long getId() {
+    /**
+     * Returns id of a reservation
+     * 
+     * @return id
+     */
+    public Long getId() {
         return id;
     }
 
+    /**
+     * Sets reservation id to a new, specified value
+     * 
+     * @param id New value for id
+     */
     public void setId(long id) {
         this.id = id;
     }
 
+    /**
+     * Returns user associated with reservation
+     * 
+     * @return User
+     */
     public User getUser() {
         return user;
     }
 
+    /**
+     * Sets user associated with reservation to a new value
+     * 
+     * @param user New user for reservation
+     */
     public void setUser(User user) {
         this.user = user;
     }
 
+    /**
+     * Gets room associated with reservation
+     * 
+     * @return Room
+     */
     public Room getRoom() {
         return room;
     }
 
+    /**
+     * Sets room associated with reservation to a new value
+     * 
+     * @param room New room for reservation
+     */
     public void setRoom(Room room) {
         this.room = room;
     }
 
+    /**
+     * Gets start time of a reservation
+     * 
+     * @return Start time of a reservation
+     */
     public LocalDateTime getStartTime() {
         return startTime;
     }
 
+    /**
+     * Sets start time of a reservation
+     * 
+     * @param startTime New start time value
+     */
     public void setStartTime(LocalDateTime startTime) {
         this.startTime = startTime;
     }
 
+    /**
+     * Gets end time of a reservation
+     * 
+     * @return End time of a reservation
+     */
     public LocalDateTime getEndTime() {
         return endTime;
     }
 
+    /**
+     * Sets end time of a reservation
+     * 
+     * @param endTime New end time value
+     */
     public void setEndTime(LocalDateTime endTime) {
         this.endTime = endTime;
     }
+    
+    @Override
+    public int hashCode() {
```

Comment:
To stejne co jsem psal Tomasovi k hashCode. Melo by byt implementovano pres `Objects.hash()`.